include/envoy/stream_info/stream_info.h

```diff
@@ -128,6 +128,8 @@ struct ResponseCodeDetailValues {
   // indicates that original "success" headers may have been sent downstream
   // despite the subsequent failure.
   const std::string LateUpstreamReset = "upstream_reset_after_response_started";
+  // The request was rejected due to invalid HTTP headers
```

Comment:
Specifically, the headers had invalid characters, yes?  If so let's be as specific as possible here and below